JordanKnott/taskcafe
# Fix user password

### Aim of the fix

Fix a bug on the admin page where clicking resetting password would show the 'remove from team' popup.
Fix a typo of 'organization'.
Fix the validation on the 'reset password' popup.
I've fixed a few errors that said 'key' was undefined which stopped the script from running.

### How to Test

Log in as an admin and go to the admin settings.
Click 'Reset Password' on a user. Check that the team popup no longer pops up but the reset password popup shows instead.
The 'Reset Password' should become enabled if you type a matching password in both fields.

### Notes

I've added a `onKeyUp` listener - I'm not sure if this is the best way to do this (I don't do a lot of react), so I'm happy to listen to any feedback on this. Added the `onKeyUp` instead of editing `onChange` which seemed to be used by something else.

I'm not actually sure the password reset button is hooked up to anything (again, happy for any pointers/tips here on how to fix this)
